docs/docs/Configuration/configuration-cli.mdx

```diff
@@ -72,88 +87,167 @@ The following sections describe the available CLI commands and any non-default o
 
 Running the CLI without any arguments prints a list of available options and commands:
 
+<Tabs groupId="Invocation">
+  <TabItem value="uv (recommended)" label="uv (recommended)" default>
+
+```bash
+uv run langflow
+```
+
+  </TabItem>
+  <TabItem value="Direct" label="Direct">
+
 ```bash
 langflow
-# or
-python -m langflow
 ```
 
+  </TabItem>
+</Tabs>
+
 ### langflow api-key {#langflow-api-key}
 
-Create a Langflow API key with superuser privileges.
+Create a Langflow API key.
+
+The logged-in user must be a superuser to create API keys with the CLI.
```

Comment:
I omitted this line before because we have much more detail at the link. I think most users will be a superuser if they try to run this anyway.